atividades/para-casa/bruna-andrade/contaBancaria.js

```diff
@@ -0,0 +1,53 @@
+class bankAccount {
+  constructor(name, balance, limit) {
+    this.name = name;
+    this.balance = balance;
+    this.limit = limit;
+    this.status = true;
+  }
+
+  // Define um novo limite para a conta
+  setLimit(value) {
+    if (value < 0) {
+      return "invalid value";
+    } else {
+      this.limit = value;
+    }
+  }
+
+  // Ativa ou desativa o limite da conta
+  setStatus(condition) {
+    if (condition) {
+      this.status = true;
+    } else if (!condition) {
+      this.status = false;
+    } else {
+      return "invalid value";
+    }
+  }
+
+  // Realiza um saque na conta
+  withdraw(value) {
+    let balanceAvailable = this.limit + this.balance;
+
+    if (
+      this.balance >= value ||
+      (this.balance < value && balanceAvailable >= value && this.status == true)
```

Comment:
Neste caso, não está levando em consideração quando ele usa o limite, pois no exercício diz que quando o valor do saldo é insuficiente pode ser usado o limite e o saldo fica negativo com o uso do limite.

Poderia ser usado uma nova condição para quando o limite for usado.